src/cmds/mod.rs

```diff
@@ -63,12 +63,12 @@ pub struct Cli {
 #[derive(Debug, Args)]
 pub struct SolanaArgs {
     /// Solana keypair file path
-    #[arg(long, env = ENV_SOLANA_WALLET)]
-    pub wallet: Option<PathBuf>,
+    #[arg(long, alias = "solana-keypair", env = ENV_SOLANA_KEYPAIR)]
```

Comment:
these aliases aren't necessary; clap automatically translates the command-line arg flag from dashes to the underscores in the struct field names